popup/date-since.md

```diff
@@ -0,0 +1,51 @@
+
+# Add a date since last update field to a pop-up
+
+[![time_popup](./images/timing.PNG)]
+
+This example shows how to add days since the last update to a pop-up
+
+## Use cases
+
+This is designed specifically for the popup profile. These set of expressions allow you to represent the number of days since a point or polygon has been udpated.
+
+## Workflow
+
+Copy and paste the expression found in the expression template below to the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or the relevant location in a custom app.
+
+## Expression Template
+
+Create an expression to identify the date field and find the 
+
+```js
+//get the time currently
+var timeNow = Date()
+//get the time field form your feature layer
+var survey = $feature["date_survey1"]
```

Comment:
For something like an expression template, it's more useful to place an obviously fake field name where the user needs to conform the template to their data. I typically do something like the following:

```suggestion
var survey = $feature["YOUR_DATE_FIELD_NAME_HERE"];
```